core/src/components/input/input.ionic.scss

```diff
@@ -20,6 +20,15 @@
   min-height: 48px;
 }
 
+// Input Bottom Content
+// ----------------------------------------------------------------
+
+.input-bottom {
+  @include padding-horizontal(0, 0);
+
+  font-weight: 500;
```

Comment:
While I was implementing the valid state, I noticed that some styles were missing from the helper text.

Reply:
Are you able to pull the latest changes/having the design tokens to use:
```
font-weight: $ionic-font-weight-medium;
```

?